opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -868,14 +900,22 @@ namespace cpgrid
                 // Computations in this loop could be speeded up
                 // by doing more of them simultaneously.
                 int output_face = face_to_output_face[face];
-                IndirectArray<point_t> face_pts(points, fn + fp[output_face], fn + fp[output_face+1]);
-                point_t avg = average(face_pts);
-                point_t centroid = polygonCentroid(face_pts, avg);
-                point_t normal = polygonNormal(face_pts, centroid);
-                double area = polygonArea(face_pts, centroid);
-                face_normals.push_back(normal);
-                face_centroids.push_back(centroid);
-                face_areas.push_back(area);
+                if (output_face == cpgrid::NNCFace) {
+                    // NNC faces are purely topological constructs,
+                    // and do not have any embedded geometry.
+                    face_normals.push_back({-1e100, -1e100, -1e100});
+                    face_centroids.push_back({-1e100, -1e100, -1e100});
+                    face_areas.push_back(-1e100);
```

Comment:
This will not work with the ewoms code. Why not try to use some reasonable values?

- Normal should use the two two cell centroids as direction and have length 1
- Centroid should be the midpoint between the two cells
- Area should be one to make sure it can be used with the ewoms discretization code.

Reply:
I wanted to ensure the values were not actually used, and stand by that: it is better to get crazy results when you try something wrong. However for the area I had to modify it (already in this PR) for the reason you state. I think the normal and centroid should remain invalid. Reasonable values will hide errors.